composer.json

```diff
@@ -76,13 +76,10 @@
         "drupal/address": "~1.2",
         "drupal/admin_toolbar": "^2.2",
         "drupal/allowed_formats": "~1.1",
-        "drupal/better_normalizers": "^1.0-beta3",
-        "drupal/block_blacklist": "~1.0",
```

Comment:
We use this. Is it somewhere else?

Reply:
its not in core.extension.yml so i don think we're using it.

Reply:
I am totally thinking of layout builder restrictions. 

All good.